src/librawspeed/decompressors/HasselbladDecompressor.cpp

```diff
@@ -44,7 +44,7 @@ HasselbladDecompressor::HasselbladDecompressor(const RawImage& mRaw_,
 
   // FIXME: could be wrong. max "active pixels" - "100 MP"
   if (mRaw->dim.x == 0 || mRaw->dim.y == 0 || mRaw->dim.x % 2 != 0 ||
-      mRaw->dim.x > 12000 || mRaw->dim.y > 8816) {
+      mRaw->dim.x > 12000 || mRaw->dim.y > 8842) {
```

Comment:
(nitpick) Like i said, this is adding support for uncompressed, why is it touching something else?

Reply:
I ended up not merging this part.
After the main change of this PR, no RPU sample fails with this error,
which leads me to believe this was superficial.
This is why i'm asking for small separate changes.

Reply:
In theory, the tethered download from camera will produce a compressed .FFF file of this size, but indeed, there is no such sample in RPU currently. Maybe @kkempter-getswift can supply one as well please?